src/Passwordless/PasswordlessClient.cs

```diff
@@ -14,9 +14,30 @@ namespace Passwordless;
 
 /// <inheritdoc cref="IPasswordlessClient" />
 [DebuggerDisplay("{DebuggerToString(),nq}")]
-public class PasswordlessClient(HttpClient http, bool disposeClient, PasswordlessOptions options)
+public class PasswordlessClient(HttpClient http, bool disposeHttp, PasswordlessOptions options)
     : IPasswordlessClient, IDisposable
 {
+    private static readonly string SdkVersion =
+        typeof(PasswordlessClient).Assembly.GetName().Version?.ToString(3) ??
+        // This should never happen, unless the assembly had its metadata trimmed
+        "unknown";
+
+    private readonly HttpClient _http = new(new PasswordlessHttpHandler(http, disposeHttp), true)
+    {
+        BaseAddress = new Uri(options.ApiUrl ?? PasswordlessOptions.CloudApiUrl),
+        DefaultRequestHeaders =
+        {
+            {
+                "ApiSecret",
+                options.ApiSecret
```

Comment:
Shouldn't be possible since you need to pass the options to the constructor (or the `AddPasswordlessSdk` method). If the secret isn't set, it will error out before the request is even sent.